global.css

```diff
@@ -20,9 +20,9 @@ header {
 }
 .darkmode {
   background-color: #404040;
-  color: #ffffff;
+  color: #121212;
 }
 .darkmode1 {
```

Comment:
you could rename these so it is visible what it styles - also you could think about using BEM